.travis.yml

```diff
@@ -34,12 +34,28 @@ before_script:
   # Export database variable for kernel tests.
   - export SIMPLETEST_DB=mysql://root:@127.0.0.1/og
 
+  # Composer.
+  - sed -i '1i export PATH="$HOME/.composer/vendor/bin:$PATH"' $HOME/.bashrc
+  - source $HOME/.bashrc
+
+  # Install Composer dependencies.
+  - composer self-update
+
+  # Install code sniffer.
+  - composer global require squizlabs/php_codesniffer
+
+  # Coder.
+  - composer global require drupal/coder:dev-8.x-2.x
+  - ln -s ~/.composer/vendor/drupal/coder/coder_sniffer/Drupal ~/.composer/vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/
+  - phpenv rehash
+
+  # Check if there any bad coding standarts
+  - phpcs --standard=Drupal -p --extensions=php,module,install,test,profile,theme,js,css,info $TESTDIR
```

Comment:
some extensions can be removed, so we can do: `phpcs --standard=Drupal -p --extensions=php,module,install,test,js,css,info $TESTDIR`